src/checking/issue_mapping/READ_ME_FOR DEVELOPER.txt

```diff
@@ -0,0 +1,20 @@
+	
+/** IMPORTS : maybe you need to configure the path first */
+import * as ISSUE from "./issue_mapping";
+	
+	
+	/**
+	* IF You want to add an issue use the follow code snippet:
+	*/
+	const issueJSONObjectKey = /** SELECT ISSUE PATH HERE for example: ISSUE.knownIssueCodes.WARNINGS.MISSING_STATEMENT_AT_THE_END;*/;
+	
+	const severityNumber = /** SELECT TYPE HERE: ISSUE.Severity.HINT|WARNING|ERROR|FATAL-ERROR*/;
+	const issueCode = ISSUE.getIssueCodeFromJSON(issueJSONObjectKey); /** get the code/key of the issue path OR write your own code here*/
+	const issueMessage = issueJSONObjectKey.MISSING_STATEMENT_AT_THE_END; /** use path and add the last path element to get the issue message OR write your own message here*/
+	const positionNumber = /** add a number where the issue is found in the code or add NaN here */; 
+
+	/** creates an issueObject*/
+	const issue: IssueObject = ISSUE.createIssue(severityNumber, issueCode, issueMessage, positionNumber);
+	
+	/** add issue to List*/
+	ISSUE.addIssueToIssueList(issue); 
```

Comment:
If you want to include information for the context of a PR only, you can put that in the PR-description. I you think the code of a folder needs some persistent documentation, you can rename this file `readme.md` and make use of [Markdown](https://guides.github.com/features/mastering-markdown/) syntax.